src/test/java/org/commcare/test/utilities/MockApp.java

```diff
@@ -40,13 +40,6 @@ public MockApp(String resourcePath) throws Exception {
         final LivePrototypeFactory mPrototypeFactory = setupStaticStorage();
         MockUserDataSandbox mSandbox = new MockUserDataSandbox(mPrototypeFactory);
         CommCareConfigEngine mEngine = new CommCareConfigEngine(mPrototypeFactory);
-        
-        mEngine.setStorageFactory(new IStorageIndexedFactory() {
```

Comment:
I think this is needed for tests to run properly, or is this also just the result of merging? i'm not sure I see from this PR anything that would prevent this from being needed